### PR TITLE
[JA] Translate lessons/basics/mix

### PR DESCRIPTION
**Pull request by starfish719**
I translated `lessons/basics/modules` section!
Please review! @elixirschool/japanese

refs: https://github.com/elixirschool/elixirschool/pull/1812

### CODE REVIEW

**Comment by marocchino** [comment]
https://github.com/elixirschool/elixirschool/commit/fe272f069a44c55bb0c8ab9e32fc5d6b14ce926a#diff-9a7600c780427e1cf724d5901498690e

File: ja/lessons/basics/mix.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
-version: 1.0.2
+version: 1.1.0
 title: Mix
 ---
 
-私たちがElixirの深海へと潜れるようになる前に、まずmixについて学ぶ必要があります。Rubyに詳しければ、mixとはBundlerとRubyGems、そしてRakeが組み合わさったものです。あらゆるElixirプロジェクトに欠かせない部分であり、このレッスンではその素晴らしい機能のうち、ほんの一部を探検していきます。mixが提供する全ての機能を知るには`mix help`を実行してください。
+私たちがElixirの深海へと潜れるようになる前に、まずmixについて学ぶ必要があります。Rubyに詳しければ、mixとはBundlerとRubyGems、そしてRakeが組み合わさったものです。あらゆるElixirプロジェクトに欠かせない部分であり、このレッスンではその素晴らしい機能のうち、ほんの一部を探検していきます。mixが提供する全ての機能を知るには、最新の環境で`mix help`を実行してください。
```
Comment (marocchino) [inline ja/lessons/basics/mix.md]: maybe) If the current is used with the version, it has the meaning of new, but this case does not seem to be the case.

Comment (marocchino) [reply ja/lessons/basics/mix.md]: @starfish719  Please give me a comment if you want to hold it. I'll merge.

Comment (starfish719) [reply ja/lessons/basics/mix.md]: @marocchino `in the current environment` is mean `your enviroment`.
`最新の環境` is mean `newest version`.
So, I'll fix it.

Thank you for your review.

Comment (starfish719) [reply ja/lessons/basics/mix.md]: @marocchino I fix it!
Please review.

https://github.com/elixirschool/elixirschool/pull/1872/commits/dc27c6358086c468be65279b9754cff005b8c428

**Comment by starfish719** [comment]
@marocchino Thank you for your review 👍

**Pull request merged**